rules/windows/file/file_event/file_event_win_webshell_creation_detect.yml

```diff
@@ -19,6 +20,13 @@ detection:
     selection_ext1:
         TargetFilename|contains:
             - '.asp'
+            - '.aspx'
+            - '.soap'
+            - '.jsp'
+            - '.jspx'
+            - '.jsw'
+            - '.jspf'
+            - '.jsv'
```

Comment:
These extensions are all related to JSP which can't be hosted by default in IIS. And since the current selection is targeting IIS with the `\inetpub\wwwroot\` path. These additions will not be very useful to the rule.

You need to add a new set of selections that focuses on the path used by a webserver that can leverage JSP files

Reply:
Resolving this due to unresponsiveness. See final comment below